### PR TITLE
Added BACKUP_KEEP_DAYS option.

### PR DESCRIPTION
**Pull request by fulminant**
Added backups auto-removal by chosen time.
#13

### CODE REVIEW

File: src/backup.sh [hunk 1]
```diff
@@ -83,3 +83,13 @@ aws $aws_args s3 cp "$local_file" "$s3_uri"
 rm "$local_file"
 
 echo "Backup complete."
+
+if [ "$BACKUP_KEEP_DAYS" -ne 0 ]; then
+  ms=$((86400*BACKUP_KEEP_DAYS))
```
Comment (eeshugerman) [inline src/backup.sh:88]: Why `ms`? To me that suggests milliseconds, but looks like this is seconds.

File: src/backup.sh [hunk 2]
```diff
@@ -83,3 +83,13 @@ aws $aws_args s3 cp "$local_file" "$s3_uri"
...
+  date_from_remove=$(date -d "@$(($(date +%s) - ms))" +%Y-%m-%d)
+  backups_query="Contents[?LastModified<='${date_from_remove} 00:00:00'].{Key: Key}"
+
+  echo "Removing old backup from $S3_BUCKET..."
+  aws s3api list-objects --bucket "${S3_BUCKET}" --prefix "${S3_PREFIX}" --query "${backups_query}" --output text | xargs -n1 -t -I 'KEY' aws s3 rm s3://"${S3_BUCKET}"/'KEY'
```
Comment (eeshugerman) [inline src/backup.sh:93]: Please use the --long-form xargs flags for readability (I'll admit, I had to look all these up :laughing:). Also a line break or two if you don't mind.

**Review by eeshugerman — CHANGES_REQUESTED** [review]
Thank you for contributing!

File: Dockerfile [hunk 3]
```diff
@@ -19,6 +19,7 @@ ENV S3_ENDPOINT ''
 ENV S3_S3V4 'no'
 ENV SCHEDULE ''
 ENV PASSPHRASE ''
+ENV BACKUP_KEEP_DAYS 7
```
[the remark this answers is not in the document]
Comment (fulminant) [reply Dockerfile]: Make a sense.

File: src/backup.sh (hunk 1 above)
Comment (fulminant) [reply src/backup.sh]: my bad.

File: src/backup.sh (hunk 2 above)
Comment (fulminant) [reply src/backup.sh]: For some reason long-form `xargs` doesn't work with the Alpine container(

**Review by eeshugerman — APPROVED** [review]

**Pull request merged**